### PR TITLE
feat: add Pino logger

### PR DESCRIPTION
**Pull request by mattyod**
As per other repos, enabling Pino and removing undesired console logs.

### CODE REVIEW

File: app/auth/id-token/nonce.js [hunk 1]
```diff
@@ -9,22 +9,15 @@ const generate = (request) => {
...
-      throw new Error('HTTP Session contains no nonce')
-    }
-    if (nonce !== idToken.nonce) {
-      throw new Error('Nonce mismatch')
-    }
-  } catch (error) {
-    console.log(`${new Date().toISOString()} Error while verifying id_token nonce: ${error.message}`)
-    console.error(error)
-    throw error
+  if (typeof idToken === 'undefined') {
```
Comment (RobCatton) [inline app/auth/id-token/nonce.js:12]: Should this be typeof?

File: app/routes/endemics/index.js [hunk 2]
```diff
@@ -54,13 +56,13 @@ module.exports = {
         session.setEndemicsClaim(request, referenceKey, tempClaimId)
 
         // new user
-        if ((!Array.isArray(claims) || !claims?.length) && latestVetVisitApplication === undefined) {
+        if ((claims.length === 0) && latestVetVisitApplication === undefined) {
```
Comment (RobCatton) [inline app/routes/endemics/index.js:59]: Is claims ever undefined? Just because I think it will fail on the .length if it ever can be

Comment (mattyod) [reply app/routes/endemics/index.js:59]: No we always return an array, catching 404s in the error handler at the moment but need to fix the endpoint in ffc-ahwr-application really. 

https://github.com/DEFRA/ffc-ahwr-farmer-claim/pull/404/files#diff-d41e3c3a63b911a844378e8760c201f4dd18ff5df27e2f58bf95fc700bf279eaR16

File: app/auth/id-token/nonce.js (hunk 1 above)
Comment (mattyod) [reply app/auth/id-token/nonce.js:12]: Not something I changed, but yes it does seem cumbersome assertion when `if (idToken === undefined)` would also cover it.

**Review by RobCatton — APPROVED** [review]

**Pull request merged**